notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    #Print out confirmation that we downloaded a FITS file
</pre><p>Since the download_products call prints some output, either that a new file was downloaded or a cached filed was found, is this call necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>